CHANGELOG/CHANGELOG-1.13.md

```diff
@@ -0,0 +1,89 @@
+# Changelog 1.13
+
+## 1.13.0 - 2023-07-24
+
+### Added
+
+- Add AdminNetworkPolicy support in Antrea to align with K8s NetworkPolicy API, and document the introduction and usage. ([#5170](https://github.com/antrea-io/antrea/pull/5170) [#5270](https://github.com/antrea-io/antrea/pull/5270), [@Dyanngg])
+- Add OpenAPI schema for AntreaAgentInfo and AntreaControllerInfo CRDs. ([#5206](https://github.com/antrea-io/antrea/pull/5206), [@ceclinux])
+- Support secure connections to ClickHouse DB with TLS, and allow user to define custom HTTP/HTTPs url and CA certificate. ([#5171](https://github.com/antrea-io/antrea/pull/5171), [@yuntanghsu])
+- [Flow Visibility] Add the visibility of Egress Information for Pod-to-External flows in flow records. ([#5088](https://github.com/antrea-io/antrea/pull/5088), [@dreamtalen])
+- [Flow Visibility] Adding Pod store for flow-exporter and flow-aggregator to query Pod information. ([#5185](https://github.com/antrea-io/antrea/pull/5185), [@yuntanghsu])
+- Add a new Prometheus metric to represent the number of packets dropped by OVS meter. ([#5165](https://github.com/antrea-io/antrea/pull/5165), [@mengdie-song])
+- Containerize Windows userspace OVS processes in OVS container in Antrea agent Pod to align with the design of Linux. ([#4936](https://github.com/antrea-io/antrea/pull/4936) [#5052](https://github.com/antrea-io/antrea/pull/5052) [#5071](https://github.com/antrea-io/antrea/pull/5071), [@rajnkamr] [@Atish-iaf] [@NamanAg30])
+- Add `kubeAPIServerOverride` option to antrea-controller. ([#5056](https://github.com/antrea-io/antrea/pull/5056), [@tnqn])
+- Add a configuration toggle for FlowExporter to prevent antrea agent from showing error in log. ([#5021](https://github.com/antrea-io/antrea/pull/5021), [@yuntanghsu])
+- Consume the annotation `service.kubernetes.io/topology-mode` for AntreaProxy since the old `service.kubernetes.io/topology-aware-hints` annotation has been deprecated in K8s 1.27. ([#5241](https://github.com/antrea-io/antrea/pull/5241), [@mengdie-song])
+- Add `sort-by` flag support in more antctl resources with more fields. ([#4346](https://github.com/antrea-io/antrea/pull/4346), [@jainpulkit22])
```

Comment:
Add support for the `sort-by` flag

antctl resources -> `antctl get` commands

with more fields -> for more fields